lib/rate-limiter.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const _get = require('lodash.get');
+
+exports.register = (defaultRate, routes) => {
+  const rateTable = {};
+
+  routes.forEach((route) => {
```

Comment:
Cool, I think the single loop makes this look a lot better 👍 looking at what it's actually doing now though (populating `rateTable` and returning it), this might be better suited as a `reduce` instead of a `forEach`. What do you think?